docs/NodesOverview.md-E

```diff
@@ -0,0 +1,1114 @@
+# Node Overview
```

Comment:
I guess this file should be removed

Reply:
ah, yeah. You have any idea what is causing those files to appear?

Reply:
Maybe this https://github.com/rectorphp/rector/commit/337cbd3ad470833f909b1434f49bc326be93121e ?

Reply:
the adding was accidental

found the reason the files appeared:
https://github.com/rectorphp/rector/pull/2318